### DISCUSSION TITLE
How do I ensure Project for qodana-dotnet build with the Release Configuration

### DISCUSSION THREAD

**Discussion by alfkonee** [JetBrains/Qodana#147] [Q&A] [answered]
I'd like to ask if it's possible to make qodana-dotnet run the analysis with the Release Build Configuration instead of the default Debug Configuration which it seems to currently be using

**Comment by brichbash** [comment, marked as answer]
Please follow [QD-4779](https://youtrack.jetbrains.com/issue/QD-4779/Qodana.Net-BuildConfigurations-Support) for updates and detailed information
